component/common/loki/client/client.go

```diff
@@ -47,7 +47,7 @@ const (
 
 var Reasons = []string{ReasonGeneric, ReasonRateLimited, ReasonStreamLimited, ReasonLineTooLong}
 
-var UserAgent = fmt.Sprintf("promtail/%s", build.Version)
+var UserAgent = fmt.Sprintf("GrafanaAgent/%s", build.Version)
```

Comment:
Since we're changing the user agent here, this should use the build package from `github.com/grafana/agent/pkg/build`, _not_ `github.com/grafana/loki/pkg/util/build` like is being imported now,

Reply:
Onnit